_posts/2023-05-02-cockpit-291.md

```diff
@@ -0,0 +1,104 @@
+---
+title: Cockpit 291
+author: garrett
+date: '2023-05-02'
+tags: cockpit, machines, podman
+slug: cockpit-291
+category: release
+summary: ''
+---
+
+Cockpit is the [modern Linux admin interface](https://cockpit-project.org/).
+We release regularly.
+
+Here are the release notes from Cockpit 291, cockpit-machines 289, and cockpit-podman 68:
+
+
+## Update to patternfly 5 alpha
+
+State: **merged** @ <https://github.com/cockpit-project/cockpit/pull/18672>
+{:.note}
```

Comment:
Please link to blog post https://cockpit-project.org/blog/patternfly-v5-alpha.html